### PR TITLE
feature: add signature request scenarios tests

### PR DESCRIPTION
**Pull request by ost-ptk**
_**Make sure to fill in all the below sections.**_

## Description

- added signature request scenarios tests
- disabled Firefox CI tests
- fixed flaky onboarding test
- minor improvements 

## Linked tickets

#177 

## Checklist

- [x] Make sure this PR title follows semantic release conventions: <https://semantic-release.gitbook.io/semantic-release/#commit-message-format>

- [ ] If the PR adds any new text to the UI, make sure they are localized

- [ ] Include a screenshot or recording if implementing significant UI or user flow change

- [ ] When this PR affects architecture changes wait for review from Piotr before merging

### CODE REVIEW

File: e2e/constants.ts [hunk 1]
```diff
@@ -6,9 +6,21 @@ export const ACCOUNT_NAMES = {
...
 
-export const TIMEOUT = {
-  '15sec': 1000 * 15,
-  '50sec': 1000 * 50
+export const DEFAULT_ACCOUNT = {
+  accountName: ACCOUNT_NAMES.defaultAccountName,
+  publicKey:
+    '02021006f7e7ecba9dda1fbd68bd88b0b509ce07a24c4ce8a96c62d44bec4beb9f9d',
+  truncatedPublicKey: '02021...b9f9d'
 };
```
Comment (piotrwitek) [inline e2e/constants.ts:21]: That's good improvement

File: e2e/tests/popup/signature-request-scenarios/signature-request-scenarios.test.ts [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        await driver.wait(until.alertIsPresent());
+
+        // Store the alert in a variable
+        const alert = await driver.driver.switchTo().alert();
+
+        const alertText = await alert.getText();
+
+        assert.match(alertText, /Sign successful/);
+        await alert.accept();
+      });
```
Comment (piotrwitek) [inline e2e/tests/popup/signature-request-scenarios/signature-request-scenarios.test.ts:151]: Assertions are looking great!

**Review by piotrwitek — DISMISSED** [review]
So far PR looking great!

**Comment by piotrwitek** [comment]
@ost-ptk ok thanks, let me know when ready to merge

**Comment by ost-ptk** [comment]
@piotrwitek 
Done

**Review by piotrwitek — APPROVED** [review]

**Pull request merged**